src/components/ui/OrganizationHeader/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import Icon from 'components/ui/Icon';
+import Logo from 'components/ui/Logo';
+import Avatar from 'components/ui/Avatar';
+
+export default function OrganizationHeader() {
+  const sampleImage =
+    'https://images.unsplash.com/photo-1535713875002-d1d0cf377fde?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80';
```

Comment:
This should have actually been passed as a prop since we want it to change

Reply:
You can use this url when creating a sample image though

Reply:
I have added the prop